src/index.tmpl

```diff
@@ -1,10 +1,19 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title><%=htmlWebpackPlugin.options.title %></title>
-  </head>
-  <body>
-    <div id="app">React Application placeholder</div>
-  </body>
-</html>
+	<head>
```

Comment:
Do not use tabs, use 2 spaces instead to better look and to support many level indentation.